galley/pkg/config/event/selector.go

```diff
@@ -15,41 +15,123 @@
 package event
 
 import (
+	"fmt"
+
 	"istio.io/istio/galley/pkg/config/collection"
 	"istio.io/istio/galley/pkg/config/scope"
 )
 
-// Selector distributes events to multiple different handlers, based on collection name.
-type Selector struct {
-	handlers map[collection.Name]Handler
+// Selector distributes events to different handlers, based on collection name.
```

Comment:
The name `Selector` doesn't match the definition. Something like `Router` would be more appropriate.